src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -346,6 +352,33 @@ public InputStream playback(Integer transformedMediaId) {
         return downloadOrPlayback(transformedMediaId, AuditActivity.AUDIO_PLAYBACK, AudioRequestType.PLAYBACK);
     }
 
+    @Override
+    public List<SearchTransformedMediaResponse> searchForTransformedMedia(SearchTransformedMediaRequest getTransformedMediaRequest) {
+
+        List<TransformedMediaEntity> mediaEntities = null;
+        if (getTransformedMediaRequest != null) {
```

Comment:
Can it be null? Might be worth double checking what the screens will be sending, to ensure they aren't sending the body, but with each parameter set to null.

Reply:
Great point. Added the following to the open api spec:-

requestBody:
  required: true